### ISSUE TITLE
win10 tessbridge.cpp:5:10: fatal error: tesseract/baseapi.h: No such file or directory

### ISSUE THREAD

**Issue by johnsonz** [otiai10/gosseract#158]
> This file is generated based on `ISSUE_TEMPLATE.md`. The issue reporter must read and remove this block before submitting.

# Summary

- Installation failed on win 10 x64 by `go get -t github.com/otiai10/gosseract`
![Capture](https://user-images.githubusercontent.com/2533629/57833496-b7bb3f80-77ec-11e9-8cef-c26c05db7c1f.JPG)

```
C:\Users\user>go get -t github.com/otiai10/gosseract
# github.com/otiai10/gosseract
tessbridge.cpp:5:10: fatal error: tesseract/baseapi.h: No such file or directory
 #include <tesseract/baseapi.h>
          ^~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

# Reproducibility
`go get -t github.com/otiai10/gosseract`

## Reproducility Frequency

- 100%

## Reproducible Dockerfile

```dockerfile
FROM your-os:your-version
# Describe how to reproduce your problem
# on your environment
```

Otherwise, describe how to reproduce

1. `foo bar`
2. `spam ham`
3. `hoge fuga`

# Environment

```
uname -a
```

```
`go env`
C:\Users\user>go env
set GOARCH=amd64
set GOBIN=
set GOCACHE=C:\Users\user\AppData\Local\go-build
set GOEXE=.exe
set GOFLAGS=
set GOHOSTARCH=amd64
set GOHOSTOS=windows
set GOOS=windows
set GOPATH=D:\Go
set GOPROXY=
set GORACE=
set GOROOT=C:\Go
set GOTMPDIR=
set GOTOOLDIR=C:\Go\pkg\tool\windows_amd64
set GCCGO=gccgo
set CC=gcc
set CXX=g++
set CGO_ENABLED=1
set GOMOD=
set CGO_CFLAGS=-g -O2
set CGO_CPPFLAGS=
set CGO_CXXFLAGS=-g -O2
set CGO_FFLAGS=-g -O2
set CGO_LDFLAGS=-g -O2
set PKG_CONFIG=pkg-config
set GOGCCFLAGS=-m64 -mthreads -fmessage-length=0 -fdebug-prefix-map=C:\Users\JOHNSO~2\AppData\Local\Temp\go-build672812758=/tmp/go-build -gno-record-gcc-switches
```
`go version`
```
C:\Users\user>go version
go version go1.12.5 windows/amd64
```
```
tesseract --version
```
```
C:\Users\user>tesseract --version
tesseract v4.0.0.20190314
 leptonica-1.78.0
  libgif 5.1.4 : libjpeg 8d (libjpeg-turbo 1.5.3) : libpng 1.6.34 : libtiff 4.0.9 : zlib 1.2.11 : libwebp 0.6.1 : libopenjp2 2.2.0
 Found AVX2
 Found AVX
 Found SSE
```

**Comment by otiai10** [comment]
Is this duplicated with #132 ?

**Comment by reformc** [comment]
I have the same problem,

**Comment by otiai10** [comment]
closing due to no response

**Comment by yangjiafei** [comment]
Win11, after a day's and last night's all-night troubleshooting, I finally figured out the problem and solved it. I was using this package: github.com/otiai10/gosseract
When executing, I looked at the source code and it was the .cpp file that had been configured earlier with these two configurations:

go env -w CGO_CFLAGS=-ID:/vcpkg/installed/x64-windows/include
 go env CGO_LDFLAGS=-LD:/vcpkg/installed/x64-windows/lib -ltesseract55 -lleptonica-1.84.1

Actually, CGO_CFLAGS is configured for C language files, and should be configured in case of C++ files:
go env -w CGO_CPPFLAGS=-ID:/vcpkg/installed/x64-windows/include

1, first of all I used an exe install with no .h files at all.
2, so I searched for a long time and used the vcpkg install.
3, and found that it still reported an error. But tesseract can run on its own. So I think it's a problem with go not being able to access the path. And of course I configured the windows environment variable PATH.
